web/client/api/WFS.js

```diff
@@ -12,6 +12,13 @@ import requestBuilder from '../utils/ogc/WFS/RequestBuilder';
 import {toOGCFilterParts} from '../utils/FilterUtils';
 import { getDefaultUrl } from '../utils/URLUtils';
 
+/**
+* @note from this ref: https://www.npmjs.com/package/url#urlformaturlobj
+* * in using [ url.format(urlObj) ] --> 'urlObj.search' will be used in place of 'urlObj.query'
+* * and 'urlObj.query' (object; see querystring) will only be used if 'urlObj.search' is absent.
+* * so in case adding new parameters or update existing ones in 'urlObj.query', we need to set 'urlObj.search' with undefined to ignore it
+*/
+
```

Comment:
```suggestion
```

I don't think this comment is needed as it is not a new feature. Inline comment on the respective functions is fine